documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -373,54 +373,118 @@ For more information, see xref:mysql-schema-history-topic[schema history topic].
+  During the scan, the connector completes the following tasks:
+
+. Confirms that the table being scanned was created before the snapshot began.
+   If the table was created after the snapshot began, the connector skips the table.
+   After the snapshot is complete, and the connector transitions to streaming, it emits change events for any tables that were created after the snapshot began.
+. Produces a `read` event for each row that is captured from a table.
+   All `read` events contain the same binlog position, which is the position that was obtained in step 5.
+. Emits each `read` event to the Kafka topic for the source table.
+. Releases data table locks, if applicable.
 
 |9
-a|Records the completed snapshot in the connector offsets.
+|Commit the transaction.
+
+|10
+|Record the successful completion of the snapshot in the connector offsets.
 
 |===
 
-Connector restarts::
-If the connector fails, stops, or is rebalanced while performing the _initial snapshot_, then after the connector restarts, it performs a new snapshot. After that _intial snapshot_ is completed, the {prodname} MySQL connector restarts from the same position in the binlog so it does not miss any updates.
-+
-If the connector stops for long enough, MySQL could purge old binlog files and the connector's position would be lost. If the position is lost, the connector reverts to the _initial snapshot_ for its starting position. For more tips on troubleshooting the {prodname} MySQL connector, see xref:mysql-when-things-go-wrong[behavior when things go wrong].
+The resulting initial snapshot captures the current state of each row in the captured tables.
+From this baseline state, the connector captures subsequent changes as they occur.
 
-Global read locks not allowed::
-Some environments do not allow global read locks. If the {prodname} MySQL connector detects that global read locks are not permitted, the connector uses table-level locks instead and performs a snapshot with this method. This requires the database user for the {prodname} connector to have `LOCK TABLES` privileges.
-+
-.Workflow for performing an initial snapshot with table-level locks
+After the snapshot process begins, if the process is interrupted due to connector failure, rebalancing, or other reasons, the process restarts after the connector restarts.
+
+After the connector completes the initial snapshot, it continues streaming from the position that it read in Step 5 so that it does not miss any updates.
+
+If the connector stops again for any reason, after it restarts, it resumes streaming changes from where it previously left off.
+
+If the connector stops for long enough, the binlog files that record the connector's position can expire.
+If the database purges the expired binlog files, the connector's position is lost, which causes the connector to revert to the _initial snapshot_ for its starting position.
```

Comment:
I feel this sentence is a bit misleading.  By default, if the position in the logs points a place that is no longer available because the logs have been pruned, then the connector will fail to start throwing an error that a new snapshot must be taken.  Setting `snapshot.mode` to `when_needed` can circumvent this failure, but requires being explicitly configured.